migrator/setup.cfg

```diff
@@ -0,0 +1,49 @@
+
+[metadata]
+name = zenodo-rdm-migrator
+version = 1.0.0
+description = ZenodoRDM migrator module.
+long_description = file: README.md
+license = MIT
+author = CERN
+author_email = info@zenodo.org
+platforms = any
+url = https://github.com/inveniosoftware/zenodo-rdm/migrator
+classifiers =
+    Development Status :: 5 - Production/Stable
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=3.9
+zip_safe = False
+install_requires =
+    idutils>=1.2.1
+    invenio-rdm-migrator>=1.0.0a12,<2.0.0
+    nameparser>=1.1.1
+    kafka-python>=2.0.2
+    gssapi>=1.8.2
+    zenodo-legacy
+
+[options.extras_require]
+tests =
+    dictdiffer>=0.9.0
+    pytest-invenio
+    pytest-black
+
+[options.entry_points]
+# flask.commands =
```

Comment:
remove unused?
```suggestion

```

Reply:
Not sure if we'll eventually make this a standalone CLI and not run via `python -m zenodo_rdm_migrator` so left it in for now. Let's discuss, though it's a minor point.